clients/vscode/src/InlineCompletionProvider.ts

```diff
@@ -211,4 +211,28 @@ export class InlineCompletionProvider extends EventEmitter implements InlineComp
     // await not required
     this.client.telemetry.postEvent(params);
   }
+
+  /**
+   * Get current display range with the current completion item
+   * @return {Range | undefined} - The range with the current completion item
+   */
+  public getCurrentDisplayRange(): Range | undefined {
+    const item = this.displayedCompletion?.completions.items[this.displayedCompletion.index];
+    const range = item?.range;
+    if (!range) {
+      return undefined;
```

Comment:
I left a note here. It's not necessary to fix it in the PR.
```suggestion
      // FIXME: If the item has a null range, we can use current position and text length to calculate the result range
      return undefined;
```

Reply:
I left a note here. It's not necessary to fix it in the PR.